---
layout: post
title: "What is the functional diversity of the proteome that is important for thermal tolerance?"
date: 2016-01-26
---
Updated: 2016-06-09

<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.

Many studies have characterized how genes turn on in order to meet environmental challenges. However, little is known about the functional diversity of proteins within and among species, even though they carry out important biological activities for organisms! To approach this problem, I'll use new techniques in LC-MS/MS to figure out the stability (which is related to biological activity) across proteins I can identify.

<img src="/assets/20160126_mass_spec_proteomics.jpg" />

Schematic of methods using <a href="https://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html">Tandem Mass Tag (TMT) System</a>

Do more thermally tolerant species have more stable proteins on average? Theory predicts that this should not be the case because evolving beneficial amino acid changes is "hard"! Or is it a subset? If it is subset, then which ones are functionally enriched? And given protein sequence information, can we make informed predictions about the determinants of what makes it stable!? This has nice implications for rational drug design. 

Please check out my <a href="http://adnguyen.github.io/assets/03_ANBE_ddig_project_description_v8.pdf">DDIG, pg 5-8</a> for more details. 


Update: 2016-05-20, this project is in progress! Check our [github repo if interested](https://github.com/adnguyen/2016_Protein_stability_evolution)